src/app/bookshelf/stats/BookshelfStats.module.css

```diff
@@ -0,0 +1,62 @@
+.statsContainer {
+  margin: 2rem 4rem 2rem 4rem;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 2rem;
+  flex: 1;
+}
+
+.flex {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 2rem;
+  margin-bottom: 2rem;
+}
+
+.flexColumn {
```

Comment:
In this case, I'd suggest naming the class just '.column'. Generally, try to avert using implementation details in class names, in this case - `flex`, as the structure may stay as it is, but the styling may change (and then - if the column does not use `display: flex;` anymore for some reason, should it be named like that?)